druid/src/widget/button.rs

```diff
@@ -175,7 +177,16 @@ impl<T: Data> Widget<T> for Button<T> {
             .inset(-stroke_width / 2.0)
             .to_rounded_rect(env.get(theme::BUTTON_BORDER_RADIUS));
 
-        let bg_gradient = if is_active {
+        let bg_gradient = if ctx.is_disabled() {
```

Comment:
I don't find this particularly communicates "disabledness"; what's most important here for me is dimming the text color.

(I acknowledge that the current widget styling wasn't really designed with a concept of "disabledness",  so it's hard to make it fit)